### DISCUSSION TITLE
My haxophone - California Dreamin'

### DISCUSSION THREAD

**Discussion by mikenturner** [cardonabits/haxo-hw#34] [Show and tell]
Here's a video of me playing California Dreamin' on my haxophone
[https://youtu.be/zgSnAyciqs8](https://youtu.be/zgSnAyciqs8)
Mike

**Comment by jcard0na** [comment]
Oh, wow, Mike!  Very nicely done!  Your yellow/gray (or should I say brass/nickel-plated?) color combination is very striking!
One question:  what size of keycaps have you chosen for the wide low Bb and G# keys?  I've tried 1.25U, 1.5U and 1.75U and seem to prefer the smallest.  How about you?

**Reply by mikenturner** [reply]
Thanks Javier.

The yellow "brass" PCB was a deliberate choice, but the keycaps were just a cheap set I bought with no particular thought to colour.  The keys are all slightly different angles & heights so not ideal and I might look for a better set.  I have wider sizes for G# & Bb but I've not really experimented yet, I'm not sure it makes much difference and I may go back to normal sized ones. 
<img src="https://github.com/cardonabits/haxo-hw/assets/60625283/41bf174b-62d5-4b0f-b986-bb59f120f1bd" width = "150">

But I have used the different heights to try and make palm D more prominent, but again I probably need to experiment more.
<img src="https://github.com/cardonabits/haxo-hw/assets/60625283/43434839-5a72-4a51-8dd6-89ed92b46ac5" width = "150">

Mike

**Reply by jcard0na** [reply]
Nice!  Thanks for sharing.  It's very gratifying to see creative new ways to extend/personalize the Haxophone.
 
Regarding keycaps, I'm planning to try [these out](https://mykeyboard.eu/search/?q=%22keyreative+blank%22).  The price is good, and being in the EU should mean fast shipping.  They seem to have some issues with UK orders, though.

Alternatively you can order 1U sizes of many colors [here](https://www.aliexpress.com/item/1005003894469722.html), if you are willing to wait for very long shipping times...

Although, I'm not sure I'd change that lovely gray`Capslock` `G#` 😂!

**Reply by mikenturner** [reply]
Thank Javier - as you said, the first link says "UK Orders below 135 GBP are on hold until further notice.", so I might try the AliExpress ones as it's not urgent
Mike
